### PR TITLE
Fix/deleting for verified user

### PR DESCRIPTION
**Pull request by Sorizen**

### CODE REVIEW

File: src/common/modal/RefreshModal.vue [hunk 1]
```diff
@@ -234,7 +235,13 @@ const checkIsSubmoduleLinkValid = async () => {
           ? { module_name: currentModuleId.value }
           : {}),
         ...(submoduleToSearch.value
-          ? { submodule: submoduleToSearch.value }
+          ? {
+              submodule:
+                submoduleToSearch.value &&
```
Comment (KlausKidman) [inline src/common/modal/RefreshModal.vue]: remove `submoduleToSearch.value`

File: src/pages/unverified-users/UnverifiedUsersList.vue [hunk 2]
```diff
@@ -132,21 +131,14 @@ const getUnverifiedUsersList = async () => {
...
+})
 
-watch(
-  () => [props.searchText, currentModuleFilter],
-  async () => {
-    currentPage.value = 1
-    await getUnverifiedUsersList()
-  },
-)
+watch([() => props.searchText, currentModuleFilter], async () => {
```
Comment (KlausKidman) [inline src/pages/unverified-users/UnverifiedUsersList.vue]: remove `async`

File: src/common/modal/RefreshModal.vue (hunk 1 above)
Comment (Sorizen) [reply src/common/modal/RefreshModal.vue]: done

File: src/pages/unverified-users/UnverifiedUsersList.vue (hunk 2 above)
Comment (Sorizen) [reply src/pages/unverified-users/UnverifiedUsersList.vue]: done

**Review by KlausKidman — APPROVED** [review]

**Review by KlausKidman — APPROVED** [review]

**Pull request merged**